### PR TITLE
DEVX-1524: Adds a protobuf example configuration

### PR DESCRIPTION
**Pull request by rspurgeon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,12 @@ Parameter | Description | Default
 
 See the [config](https://github.com/confluentinc/kafka-connect-datagen/tree/master/config) folder for sample configurations. 
 
+## Supported data formats
+
+Kafka Connect supports [Converters](https://docs.confluent.io/current/connect/userguide.html#connect-configuring-converters) which can be used to convert record key and value formats when reading from and writing to Kafka. As of the 5.5 release, Confluent Platform shipped JSON and Protobuf converters to go along with the existing Avro converter.
```
Comment (ybyzek) [inline README.md:136]:
```suggestion
Kafka Connect supports [Converters](https://docs.confluent.io/current/connect/userguide.html#connect-configuring-converters) which can be used to convert record key and value formats when reading from and writing to Kafka. As of the 5.5 release, Confluent Platform packages Avro, JSON, and Protobuf converters (earlier versions package just Avro converters).
```

File: README.md [hunk 2]
```diff
@@ -131,6 +131,12 @@ Parameter | Description | Default
...
+
+For an example of using the the Protobuf converter with kafka-connect-datagen, see this [example configuration](config/connector_users_protobuf.config). Take note of the required use of the `SetSchemaMetadata` [Transformation](https://docs.confluent.io/current/connect/transforms/index.html) to address a known issue with compatibility of schema names used by kafka-connect-datagen and Protobuf.
```
Comment (ybyzek) [inline README.md:138]:
```suggestion
For an example of using the the Protobuf converter with kafka-connect-datagen, see this [example configuration](config/connector_users_protobuf.config). Note the use of the `SetSchemaMetadata` [Transformation](https://docs.confluent.io/current/connect/transforms/index.html) which addresses a compatibility issue between schema names used by kafka-connect-datagen and Protobuf.
```

Comment (ybyzek) [reply README.md:138]: @rspurgeon please elaborate on the issue -- just a 5 word summary.  

FYC also submit a GH issue that captures the issue in more detail and link here

File: config/connector_users_protobuf.config [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "datagen-protobuf-users",
+  "config": {
+    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+    "kafka.topic": "users",
+    "quickstart": "users",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "io.confluent.connect.protobuf.ProtobufConverter",
+    "value.converter.schemas.enable": "false",
+    "value.converter.schema.registry.url": "http://localhost:8081",
+    "producer.interceptor.classes": "io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor",
```
Comment (ybyzek) [inline config/connector_users_protobuf.config:11]: FYIC remove interceptors

Comment (rspurgeon) [reply config/connector_users_protobuf.config:11]: I created this https://github.com/confluentinc/kafka-connect-datagen/issues/63, which I'll also track internally.  Considering there are other examples, I'll address this seperately

**Pull request merged**